multicluster/config/overlays/leader-ns/coverage/manager_command_patch_coverage.yaml

```diff
@@ -7,7 +7,11 @@ spec:
   template:
     spec:
       containers:
-        - command: ["/bin/sh"]
-          args: ["-c", "/antrea-mc-controller-coverage -test.run=TestBincoverRunMain -test.coverprofile=antrea-mc-controller.cov.out leader --config=/controller_manager_config.yaml; while true; do sleep 5 & wait $!; done"]
-          name: antrea-mc-controller
+        - args:
+          - /antrea-mc-controller-coverage
```

Comment:
I checked the failure in the test cluster with following errors:
```
      Message:      failed to create containerd task: failed to create shim task: OCI runtime create failed: runc create
failed: unable to start container process: exec: "/antrea-mc-controller": stat /antrea-mc-controller: no such file or
directory: unknown
```
It should be caused by a stale cluster in the testbed. Last week, I changed the definition of the MC job and forgot to reset it back.  So the deployment merged old manifest with new changes in this PR. I just rerun the test, let's wait for the new result.

Reply:
seems that the job is still failing

Reply:
The error is still the same as I posted, the reason should be that we didn't define the `command` in this PR, and it doesn't patch the command part and keep the same as before when the final manifest is generated.
Please keep the line 10 the same as before '- command: ["/bin/sh"]'
And you should do the same as other mc yaml files. Btw, you should keep the line `name: antrea-mc-controller`.

Reply:
I don't think this is right. There should be no `command` in the YAML manifest, the Dockerfile defines the correct ENTRYPOINT.

Reply:
I think ideally the patch(es) we apply would _remove_ the `command` attribute (which is not possible with a strategic merge patch).

BTW I also realized that the patches for multicluster coverage are not correct: I don't think the `GOCOVERDIR` variable is added correctly.

@shikharish it's pretty straightforward to generate the coverage manifests locally to verify them. For example, you can use:
```
./multicluster/hack/generate-manifest.sh -l antrea-multicluster -c > /tmp/leader-manifest.yml
```

Reply:
Fixed.